### PR TITLE
[bitnami/*] Fix values.yaml for internal automations

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**
Standardise `values.yaml` that uses readmenator to the format supported by our internal automations

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/wordpress/values.yaml [hunk 1]
```diff
@@ -14,9 +14,6 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
-  ## E.g.
-  ## storageClass: myStorageClass
-  ##
```
Comment (juan131) [inline bitnami/wordpress/values.yaml:19]: Why is this required?

Comment (miguelaeh) [reply bitnami/wordpress/values.yaml:19]: Our internal processes only match the first occurrence. Commented or not, so if we add both commented and no commented it will duplicate the line

**Review by carrodher — APPROVED** [review]
Lgtm

File: bitnami/wordpress/values.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/wordpress/values.yaml:19]: Then, this is a limitation to be solved in the internal process, not here.

By doing this, the final product (the chart in this case) is the one "harmed". I know it's harder to fix this in the internal automations but that's not an excuse since the product shouldn't be adapted to fit in the pipeline but the other way around.
